CHANGES.md

```diff
@@ -51,7 +51,7 @@ Issues Resolved
  * 1901 Make nunitlite-runner Prefer32Bit option consistent across Debug/Release
  * 1904 Add .NET Standard 1.6 Dependencies to the Nuspec Files
  * 1907 Handle early termination of multiple assert block
- * 1911 Changing misleading comment that implies that every ICollection<T> is a list
+ * 1911 Changing misleading comment that implies that every ICollection&lt;T> is a list
```

Comment:
Odd that we never noticed that the `<` were getting eaten in the past. Good catch.

Might I suggest that since every use in this file is for a generic type, we convert 
```md
ICollection<T> to `ICollection<T>` instead of ICollection&lt;T>
```

Trying here, `ICollection<T>`.

This will make it much more readable as plain text.